docs/guidance/map/unsuccessful_processes.md

```diff
@@ -10,9 +10,9 @@ In OCDS, the `ocid` is the unique identifier for a contracting process. As the i
 
 In most jurisdictions, if a procedure is cancelled or unsuccessful, and a **new procedure** is started to procure the same items, the two procedures are considered two **different** contracting processes. This is in keeping with the OCDS definition of a contracting process.
 
-But in other jurisdictions, such as Paraguay, the planning stage is considered as the initiation of the process. In these jurisdictions when a tender fails and a new tender is started, the two tenders are considered part of the same contracting process. This differs from the OCDS definition of a contracting process.
+But in other jurisdictions, such as Paraguay, the planning is considered as the initiation of the process. In these jurisdictions when a tender fails and a new tender is started, the two tenders are considered part of the same contracting process. This differs from the OCDS definition of a contracting process.
 
-In OCDS, it is relevant and desirable to include the planning information that relates to the process, but the contracting process is not interpreted as ‘starting’ with the planning stage. In OCDS, the planning stage is something that comes **before** the initiation of a contracting process. The initiation of the procedure is not the planning stage, because at least one of the following is true of a planning stage: it is not a concrete attempt to award one or more contracts like a request for tender, etc.; it is not a concrete opportunity for potential suppliers to participate in; it does not describe the competitive conditions.
+In OCDS, it is relevant and desirable to include the planning information that relates to the process, but the contracting process is not interpreted as ‘starting’ with the planning. In OCDS, the planning process is something that comes **before** the initiation of a contracting process. The initiation of the procedure is not the planning process, because at least one of the following is true of a planning process: it is not a concrete attempt to award one or more contracts like a request for tender, etc.; it is not a concrete opportunity for potential suppliers to participate in; it does not describe the competitive conditions.
```

Comment:
```suggestion
In OCDS, it is relevant and desirable to disclose information about planning, but the contracting process is not interpreted as "starting" with planning. In OCDS, the planning process is something that comes **before** the initiation of a contracting process. The initiation of the procedure is not the planning process, because at least one of the following is true of a planning process: it is not a concrete attempt to award one or more contracts like a request for tender, etc.; it is not a concrete opportunity for potential suppliers to participate in; it does not describe the competitive conditions.
```